### DISCUSSION TITLE
Firmware for hero 5 Black

### DISCUSSION THREAD

**Discussion by JosickG** [gopro/labs#1030] [Feature Requests]
I'm looking for the labs firmware for my old Hero 5 
On the firmware page all I find is a firmware for Hero 5 session, which I tried but does not install.
The other closest availble is for Hero 7 Black and I do not dare to test it, not to freeze the camera
Does anyone have an idea?

**Comment by dnewman-gpsw** [comment]
There is no Labs firmware for HERO5 or 6.  HERO5 Session is not really Labs, only had the most basic features.  Labs really started with HERO8.

**Comment by JosickG** [comment]
Thanks
Do you think I could test the firmware for 7 or 8?

**Reply by dnewman-gpsw** [reply]
On a HERO 7 or 8 camera only.  The hardware of HERO5 has no similarity.  Firmware is designed for each camera model, they do not run on other cameras.

**Comment by JosickG** [comment]
OK, Thanks
